src/recensio/plone/behaviors/configure.zcml

```diff
@@ -3,4 +3,12 @@
     xmlns:plone="http://namespaces.plone.org/plone"
     >
 
+  <plone:behavior
+      name="recensio.authors"
+      title="Authors Behavior"
+      factory=".authors.Authors"
+      provides=".authors.IAuthors"
+      for="plone.dexterity.interfaces.IDexterityContent"
```

Comment:
```suggestion
```

Reply:
Plone does like I did and I do like Plone: https://github.com/plone/plone.app.dexterity/blob/b46fd64b8a76988d49bee10938eb318f6d302fbd/plone/app/dexterity/behaviors/configure.zcml#L13-L20